arboretum/permissions/fetchers/github/fetch_org_collaborators.py

```diff
@@ -0,0 +1,59 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Github organization collaborators fetcher."""
+
+import json
+from urllib.parse import urlparse
+
+from compliance.evidence import DAY, RawEvidence, raw_evidence
+from compliance.fetch import ComplianceFetcher
+from compliance.utils.services.github import Github
+
+
+class GithubOrgCollaboratorsFetcher(ComplianceFetcher):
+    """Fetch collaborators from GH organization repositories."""
+
+    def fetch_gh_org_collaborators(self):
+        """Fetch collaborators from GH organization repositories."""
+        for org in self.config.get('org.permissions.org_integrity.orgs'):
+            parsed = urlparse(org['url'])
+            gh_org = parsed.path.strip('/')
+            file_prefix = gh_org.lower().replace('-', '_')
+            base_url = f'{parsed.scheme}://{parsed.hostname}'
+            github = Github(base_url=base_url)
+            repos = github.paginate_api(f'orgs/{gh_org}/repos')
+            affiliations = (
+                org['collaborator_types']
+                if 'collaborator_types' in org else ['all']
+            )
```

Comment:
"all" should be a constant added to the common.constants module (with a comment describing its purpose).  This is something we agreed to with the auditors.  Suggest:

```suggestion
            affiliations = org.get('collaborator_types', ALL_COLLABORATORS)
```